src/Core/Prefab/PlanarView.js

```diff
@@ -56,12 +56,56 @@ function PlanarView(viewerDiv, boundingbox) {
     const tileLayer = new GeometryLayer('planar');
     const initLayer = initTiledGeometryLayer(planarSchemeTile(boundingbox));
 
-    tileLayer.preUpdate = (context, layer) => {
+    function _commonAncestorLookup(a, b) {
```

Comment:
Let's wait a bit between factorizing again... I'm fine with small scope copy-paste because it's easier to delete than a wrong abstraction.